pet.el

```diff
@@ -378,19 +383,19 @@ This variable is an alist where the key is the absolute path to a
 
 Returns the path to the `pre-commit' executable."
   (and (pet-pre-commit-config)
-       (or (executable-find "pre-commit")
+       (or (executable-find "pre-commit" t)
```

Comment:
This makes PET incompatible with Emacs < 27

Reply:
The functionality that enables this PR was added in Emacs 27.

Reply:
Well we could do something like this -- thoughts?

https://github.com/joaotavora/eglot/blob/8b5532dd32b25276c1857508030b207f765ef9b6/eglot.el#L450-L453

Reply:
Exactly what I have in mind. Give it a shot, turn on GitHub Actions on your fork to see how that goes on Emacs 26 :)